docs/variables/followage.md

```diff
@@ -0,0 +1,57 @@
+---
+id: followage
+---
+
+# $(followage)
+
+Returns the time that has passed since a user followed a broadcaster.
+
+#### Parameters
+
+This variable takes **two** *optional* parameters. The **first** parameter is a **Twitch username** of a ***user*** who to fetch the follow age **of**, and the **second** parameter is a **Twitch username** of a ***broadcaster*** who to fetch the follow age **to**.
```

Comment:
The wording here seems weird to me, maybe something like ```The first parameter is the Twitch username of the user in question, and the second is the channel who they are following``` or the "target user", or something, this to me is just a bit confusing.

Reply:
I'll change it to `The **first** parameter is a **Twitch username** of a ***user*** in question, and the **second** parameter is a **Twitch username** of a ***broadcaster*** who they are following.`